files/models.py

```
@@ -1807,6 +1882,22 @@ def media_m2m(sender, instance, **kwargs):
             tag.update_tag_media()
 
 
+@receiver(m2m_changed, sender=Media.topics.through)
+def media_topics_m2m(sender, instance, action, pk_set, **kwargs):
```

Ruff (0.13.3)

Unused function argument: `sender`

(ARG001)